.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "biomejs.biome",
+    "javascript.updateImportsOnFileMove.enabled": "always",
+    "typescript.updateImportsOnFileMove.enabled": "always"
```

Comment:
I think we need to set an additional property so that VS Code doesn't change `d.ts` → `.js` when it updates imports. I don't recall what the config prop is though.

Reply:
I've no idea, but I think this behaviour is gone now, but not for sure.

Reply:
Definitely not gone. It happened to me like yesterday.